assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -44,7 +44,6 @@ const renderView = ({title, description, tabs, className}, setAttributes, isSele
           value={title}
           onChange={toAttribute('title')}
           withoutInteractiveFormatting
-          multiline="false"
```

Comment:
> wp.blockEditor.RichText multiline prop is deprecated since version 6.1 and will be removed in version 6.3. Please use nested blocks (InnerBlocks) instead. See: https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/nested-blocks-inner-blocks/

it's not necessary to migrate to `InnerBlocks` for now, since the `multiline` property is always set to `false`. But we should consider do it in the future, maybe from `6.3` and higher.

**From Worpress's core**
https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/rich-text/index.js#L115-L122

```js
	if ( multiline ) {
		deprecated( 'wp.blockEditor.RichText multiline prop', {
			since: '6.1',
			version: '6.3',
			alternative: 'nested blocks (InnerBlocks)',
			link: 'https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/nested-blocks-inner-blocks/',
		} );
	}
```

Reply:
Well the issue here is that now it is possible to have several lines of text where we wanted to limit to one line, might be it's not a big deal but that was the whole point of this prop so the behaviour will be different now 🤔

Reply:
There was also already a [ticket](https://jira.greenpeace.org/browse/PLANET-6958) for this specific update

Reply:
Actually, I've now tested it on local previous to these changes and the prop is already no longer working (due to the deprecation I guess) so we can remove it, but if we still want to be able to limit some texts to one line we'll need to think of a new solution!

Reply:
We could keep the `multiline` change out of this PR to simplify its changes/review and work on migratnig `InnerBlocks` as a follow up and part of the [PLANET-6958](https://jira.greenpeace.org/browse/PLANET-6958).

Reply:
@comzeradd moved [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1050)